lib/editing/editor.dart

```diff
@@ -45,6 +48,8 @@ abstract class Editor {
 
   bool completionAutoInvoked;
 
+  bool lookingForQuickFix;
```

Comment:
I think we can remove this field from the public API (I'll add some other comments about this).